iaso/tests/api/test_instances.py

```diff
@@ -502,6 +504,60 @@ def test_instance_create_after_sync(self):
         self.assertEqual("RDC Collecte Data DPS_2_2019-08-08_11-54-46.xml", pre_existing_instance.file_name)
         self.assertEqual("Mobile app name", pre_existing_instance.name)
 
+    def test_instance_create_with_org_unit_in_multiple_source_versions(self):
+        """POST /api/instances/ an instance with an orgunit that has been copied in multiple source versions"""
+
+        # First, let's create a user that has permissions to do all of this
+        super_yoda = self.create_user_with_profile(account=self.star_wars, username="super yoda", permissions=["iaso_sources", "iaso_submissions", "iaso_org_units"])
+
+        # Then, let's copy the existing source version through an API call (couldn't call task directly)
```

Comment:
Doing it via API call is fine, but fyi I think you should be able to call it directly with `_immediate=True`, like for example: https://github.com/BLSQ/iaso/blob/main/iaso/tests/tasks/test_process_mobile_bulk_upload.py#L137-L142